.github/workflows/aws_tfhe_tests.yml

```diff
@@ -10,17 +10,116 @@ env:
   SLACK_ICON: https://pbs.twimg.com/profile_images/1274014582265298945/OjBKP9kn_400x400.png
   SLACK_USERNAME: ${{ secrets.BOT_USERNAME }}
   SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+  IS_PULL_REQUEST: ${{ github.event_name == 'pull_request' }}
 
 on:
   # Allows you to run this workflow manually from the Actions tab as an alternative.
   workflow_dispatch:
   pull_request:
-    types: [ labeled ]
```

Comment:
Keep this line?

Reply:
Do we want to run this only on approval still ?

Reply:
good question, are the "fast tests" still there ?

Reply:
For now yes. So I guess we should keep this event types in the workflow.